src/css/components/_modules.scss

```diff
@@ -58,4 +58,5 @@
     padding-left: px-to-rem(8px);
```

Comment:
Should still use the mixin. `padding(null, null, null, px-to-rem(8px)` will render only the left.